cypress/integration/test.spec.js

```diff
@@ -0,0 +1,90 @@
+// 2개의 숫자에 대해 덧셈이 가능하다.
+// 2개의 숫자에 대해 뺄셈이 가능하다.
+// 2개의 숫자에 대해 곱셈이 가능하다.
+// 2개의 숫자에 대해 나눗셈이 가능하다.
+// AC(All Clear)버튼을 누르면 0으로 초기화 한다.
+// 숫자는 한번에 최대 3자리 수까지 입력 가능하다.
+// 계산 결과를 표현할 때 소수점 이하는 버림한다.
+
+describe('test', () => {
+    context('계산기', () => {
+        it('2개의 숫자에 대해 덧셈이 가능하다.', () => {
+            const visit = cy.visit('http://127.0.0.1:8080/');
+            const ranDigit1 = parseInt(Math.random() * 10);
+            const ranDigit2 = parseInt(Math.random() * 10);
```

Comment:
랜덤한 값을 가져오는 부분을 따로 함수로 빼도 좋을 것 같아요!
```suggestion
            // test.util.js
            export function getTestRandomValues() {
                return [parseInt(Math.random() * 10), parseInt(Math.random() * 10)]
            }
            
            const [ranDigit1, ranDigit2] = getTestRandomValues();
```